README.md

```diff
@@ -1,114 +1,235 @@
-IBM Bluemix Mobile Services - Client SDK Android Core
+BMSCore
 ===
 
 [![Build Status](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core.svg?branch=master)](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core)
-[![Build Status](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core.svg?branch=development)](https://travis-ci.org/ibm-bluemix-mobile-services/bms-clientsdk-android-core)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.ibm.mobilefirstplatform.clientsdk.android/core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.ibm.mobilefirstplatform.clientsdk.android/core)
```

Comment:
Another note: I have another task to update the README badges for all SDKs (Swift, Android, and Cordova) to be consistent. So whatever we decide will need to be done to all other SDKs for all platforms (except for platform-dependent badges like Maven Central, of course).